analysis/core/maliau_grid_definition.py

```diff
@@ -0,0 +1,80 @@
+"""Maliau grid definition.
+
+This script is used to generate
```

Comment:
This script is used to generate ??

Reply:
Yeah - got lazy on the file metadata. Will fix this once any big picture stuff has shaken down.

Reply:
Done